doc/non_dev_deployments.md

```diff
@@ -102,3 +102,12 @@ AWS key to your own, it's necessary to export the concourse password:
 ```
 export CONCOURSE_ATC_PASSWORD=the_password
 ```
+
+## Destroying an existing environment
+
+The pipelines to destroy Cloud Foundry and MicroBOSH are not setup
+automatically on non-development environments in order to prevent accidental
+deletion. They can be setup if you do need them by running:
+```
+ENABLE_DESTROY=true make <env> pipelines
+```
```

Comment:
This doesn't quite work. This adds the pipeline back in, but then when you run the destroy pipeline, the `self-update-pipeline` step deletes the pipeline while it's running.